packages/shorebird_cli/lib/src/commands/doctor_command.dart

```diff
@@ -0,0 +1,55 @@
+import 'dart:io';
+
+import 'package:mason_logger/mason_logger.dart';
+import 'package:path/path.dart' as p;
+import 'package:shorebird_cli/src/command.dart';
+import 'package:shorebird_cli/src/shorebird_version_mixin.dart';
+import 'package:shorebird_cli/src/version.dart';
+
+/// {@template doctor_command}
+///
+/// `shorebird doctor`
+/// A command that checks for potential issues with the current shorebird
+/// environment.
+/// {@endtemplate}
+class DoctorCommand extends ShorebirdCommand with ShorebirdVersionMixin {
```

Comment:
Minor but might be nice to update the shorebird_cli README to include some docs about this command.

Reply:
Good call, done